superset-frontend/packages/superset-ui-core/src/query/normalizeOrderBy.ts

```diff
@@ -56,7 +56,7 @@ export default function normalizeOrderBy(
     };
   }
 
-  // todo: Removed `legacy_ordery_by` after refactoring
+  // todo: Removed `legacy_order_by` after refactoring
   if (
     queryObject.legacy_order_by !== undefined &&
     queryObject.legacy_order_by !== null &&
```

Comment:
I checked and this was committed as a set, it looks like it was a combination of `order`+`by` twice, but w/ the final decision to be having a `_` between the words.